precompiles/warden/query_types.go

```diff
@@ -394,18 +394,23 @@ func newSignRequestsRequest(method *abi.Method, args []interface{}) (*types.Quer
 	if _, ok := types.SignRequestStatus_name[input.Status]; !ok {
 		return nil, fmt.Errorf("invalid Status value: %d", input.Status)
 	}
+	if _, ok := types.BroadcastType_name[input.BroadcastType]; !ok {
+		return nil, fmt.Errorf("invalid BroadcastType value: %d", input.BroadcastType)
+	}
 
 	return &types.QuerySignRequestsRequest{
-		Pagination: &input.PageRequest,
-		KeychainId: input.KeychainId,
-		Status:     types.SignRequestStatus(input.Status),
+		Pagination:    &input.PageRequest,
+		KeychainId:    input.KeychainId,
+		Status:        types.SignRequestStatus(input.Status),
+		BroadcastType: types.BroadcastType(input.BroadcastType),
 	}, nil
 }
 
 type signRequestsInput struct {
-	PageRequest query.PageRequest
-	KeychainId  uint64
-	Status      int32
+	PageRequest   query.PageRequest
+	KeychainId    uint64
+	Status        int32
+	BroadcastType int32
```

Comment:
This will become uint8, when you change .sol file

```suggestion
	BroadcastType uint8
```

Reply:
please read resolved comments above